feature/policy_forwarding/encapsulation/otg_tests/static_encap_gue_ipv4/README.md

```diff
@@ -0,0 +1,292 @@
+# PF-1.3: Policy-based Static GUE Encapsulation to IPv4 tunnel
+
+## Summary
+
+This test verifies Policy Forwarding (PF) action to encapsulate packets in an
+IPv4 GUE tunnel. The encapsulation is based on a statically configured GUE
+tunnel and static route.
+
+The GUE header ((UDPoIP)) used in this test refers to GUE Variant 1 as specified
+in
+[draft-ietf-intarea-gue-09](https://datatracker.ietf.org/doc/html/draft-ietf-intarea-gue-09).
+
+The following behavioral properties are called out for awareness:
+
+*   When the DSCP for the tunnel is not explicitly set, it is copied from the
+    inner to the outer encapsulating header.
+*   When the TTL for the tunnel is not explicitly set, it is decremented by 1
+    and copied from inner to the outer encapsulating header. Except TTL=1, where
+    Time Exceeded is sent back to the inner source IP.
+
+## Testbed Type
+
+*  [`featureprofiles/topologies/atedut_5.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_5.testbed)
+
+## Procedure
+
+### Test environment setup
+
+*   DUT has 5 ports connected to ATE.
+
+    ```
+                              -------
+                             |       | ==== LAG1 ==== [ ATE:Port2, ATE:Port3 ]
+          [ ATE:Port1 ] ---- |  DUT  |
+                             |       | ==== LAG2 ==== [ ATE:Port4, ATE:Port5 ]
+                              -------
+    ```
+
+*   Routes are advertised from ATE:Port2, ATE:Port3, ATE:Port4 and ATE:Port5.
+*   Traffic is generated from ATE:Port1.
+*   ATE:Port2, ATE:Port3, ATE:Port4 and ATE:Port5 are used as the destination port for GUE encapsulated traffic.
+
+#### Configuration
+
+1.  DUT:Port1 is configured as Singleton IP interface towards ATE:Port1.
+
+2.  DUT:Port2 and DUT:Port3 are configured as LAG1 IP interface towards
+    ATE:Port2 and ATE:Port3 respectively.
+
+3.  DUT:Port4 and DUT:Port5 are configured as LAG2 IP interface towards
+    ATE:Port4 and ATE:Port5 respectively.
+
+4.  DUT is configured to form two eBGP sessions with ATE using the directly
+    connected LAG interface IPs.
+
+5.  ATE:LAG1 and ATE:LAG2 are configured to advertise destination networks
+    (IPv4-DST-NET, IPv6-DST-NET) and tunnel destination (IPv4-DST-GUE) to DUT.
+    The protocol next hops for the destination networks should be re-configured
+    as below:
+
+    -   Destination network IPv4-DST-NET with protocol next hop PNH-IPv4.
+    -   Destination network IPv6-DST-NET with protocol next hop PNH-IPv6.
+
+6.  DUT is configured with an IPv4 GUE tunnel with destination IPv4-DST-GUE
+    without any explicit tunnel Type of Service (ToS) or Time to Live (TTL)
+    values.
```

Comment:
the canonical OC will show that there is no "tunnel" configuration.  There's just a static route and static NHG, NH's.  So this part can be removed.